CAM Platform/api-providers.md

```diff
@@ -3446,118 +1500,125 @@ ElasticBox-Release: 4.0
 |id | string | Instance unique identifier.|
 |schema | string | Unregistered-instance schema url.|
 
-```
-[
-  {
-    "profile": {
-      "subnet": "ca-central-1a",
-      "cloud": "vpc-083cc961",
-      "image": "ami-bf5ee2db",
-      "ebs_optimized": false,
-      "instances": 1,
-      "keypair": "Operations",
-      "security_groups": [],
-      "volumes": [],
-      "flavor": "t2.micro",
-      "schema": "http://elasticbox.net/schemas/aws/ec2/profile",
-      "managed_os": false,
-      "location": "ca-central-1"
-    },
-    "provider_id": "8c501fe3-54d7-49eb-b5d3-05016becabe3",
-    "name": "AWS LDAP Windows",
-    "created": "2017-06-15 11:40:00.243971",
-    "deleted": null,
-    "type": "Windows Compute",
-    "server_info": {
-      "public_ip": "52.62.222.233",
-      "private_ip": "172.33.22.199",
-      "public_dns": "ec2-52-62-222-233.ca-central-1.compute.amazonaws.com"
-    },
-    "updated": "2017-06-15 11:40:00.243971",
-    "power_state": "running",
-    "organization": "centurylink",
-    "external_id": "i-06bee885e4e806ca0",
-    "id": "957625f1-3c6f-423c-a8ac-430983c369f7",
-    "schema": "http://elasticbox.net/schemas/unregistered-instance"
-  },
-  {...},
-  {...}
-]
+#### Response Body
+```
+  [
+     {
+       profile: {
+       location: "us-west-2",
+       service: "cloudtrail",
+       schema: "http://elasticbox.net/schemas/aws/native-service/profile"
+       },
+       schema: "http://elasticbox.net/schemas/unregistered-instance",
+       provider_id: "8132cc1d-38b0-4867-b7a3-1af7b83d4792",
+       description: "S3 Bucket: partsultd-cloudtrail",
+       created: "2018-09-24 16:10:06.198406",
+       deleted: null,
+       type: "Native Service",
+       server_info: { },
+       updated: "2018-09-24 16:22:18.000760",
+       stack_id: null,
+       autoscaling_group: null,
+       power_state: "running",
+       organization: "mission-field",
+       external_id: "PartsUltdAPILogs-Mgmt",
+       id: "451873d3-07cb-450d-b20d-12b4ba3ec343",
+       name: "PartsUltdAPILogs-Mgmt"
+     }
+  ]
 ```
 
-**POST /services/providers/{provider_id}/images**
 
 Adds a new machine image to a provider when you give the provider ID.
 
-**Normal Response Codes**
+## POST /services/providers/{provider_id}/images
+
+Adds a new machine image to a provider when you give the provider ID.
```

Comment:
I think this is only supported for AWS and Azure; maybe that can be clarified here.